src/openforms/authentication/tests/factories.py

```diff
@@ -90,3 +97,16 @@ class RegistratorInfoFactory(factory.django.DjangoModelFactory):
 
     class Meta:
         model = RegistratorInfo
+
+
+class YiviOpenIDConnectConfigFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = YiviOpenIDConnectConfig
+        django_get_or_create = ["id"]
+
+    id = factory.Faker("id")
+
+
+class AttributeGroupFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = AttributeGroup
```

Comment:
please define a `factory.Sequence` for the `name`. It's a required and unique field.